Bottom: ffc43e5e45fdefc592b2949d27e83fa58151f313
Top:    fb5435017559ec68e2f64e4b2690185b4d090536
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 6e45042..0f6e7eb 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1256,8 +1256,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
